src/components/InfiniteScroll/index.tsx

```diff
@@ -13,13 +13,15 @@ interface InfiniteScrollProps {
 }
 
 const InfiniteScroll: SFC<InfiniteScrollProps> = ({children, dataLength, hasMore, next}) => {
+  const infiniteScrollComponentHeight = window.innerHeight - 105;
```

Comment:
This `window.innerHeight - 105;` seems hacky but it's not 🤣 

Actually, we're subtracting a specific margin from the window's height to ensure the infinite scroll component fits perfectly within the viewport. This adjustment helps avoid/prevent any additional page scroll as mentioned here: https://github.com/thenewboston-developers/thenewboston-Frontend/issues/261#issuecomment-2088084840